src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -164,7 +164,7 @@ export default class ProcessConverter extends AbstractConverter<Process> {
         shapeBpmnElement.parentId = laneId;
       } else {
         // TODO error management
-        console.log('Unable to assign lane %s as parent: flow node %s is not found', laneId, flowNodeRef);
+        console.error('Unable to assign lane %s as parent: flow node %s is not found', laneId, flowNodeRef);
```

Comment:
:question: why not putting a warning instead? Here we are just ignoring the element and so it is not displayed